.github/workflows/ci.yml

```diff
@@ -31,7 +31,7 @@ jobs:
     - name: installing dependencies
       run: npm ci
       env:
-        PHANTOMJS_CDNURL: http://cnpmjs.org/downloads
+        PHANTOMJS_CDNURL: https://dev.kiwix.org/mwoffliner/phantomjs
```

Comment:
Will this work? https://cdn.jsdelivr.net/npm/phantomjs@2.1.3/

Reply:
It does, because otherwise the CI would not be green :) right ?